RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelRawToClusterGPUKernel.cu

```diff
@@ -473,9 +476,18 @@ namespace pixelgpudetails {
     }
     __syncthreads();
 
+    if (threadIdx.x == 0) {
+      // copy the number of modules
+      nModules_Clusters[0] = *nModules;
+      // last element holds the number of all clusters
+      nModules_Clusters[1] = moduleStart[gpuClustering::maxNumModules];
+      // element 96 is the start of BPIX2 (i.e. the number of clusters in BPIX1)
+      nModules_Clusters[2] = moduleStart[96];
```

Comment:
```suggestion
      nModules_Clusters[2] = moduleStart[phase1PixelTopology::layerStart[1]];
```
to avoid proliferation of the magic number 96;
perhaps also update the comment

Reply:
on GPU this is an additional load from memory and the comment is already there.

Reply:
`layerStart` is a constexpr, is it not filled here at the compile time?
If the reason for a load needed that this is a C-array element, it may be practical to eventually take it out as a named constant, especially if the need to use `96` grows (this file already has it in 2 places).

Reply:
cuda does not accept constexpr arrays. ```layerStart``` cannot be used in device code.
I will open a issue to define single constexpr for the most used "magic" numbers in device code

Reply:
why in other contest compiles (and clang compiles as well) no clue
see
https://godbolt.org/z/WfqGvs6Gs

Reply:
and btw https://cmssdt.cern.ch/dxr/CMSSW/source/RecoLocalTracker/SiPixelClusterizer/plugins/gpuClusterChargeCut.h#89
is the only place where ```layerStart[1]``` (actually ```layerStart``` tout-court) is used in reco algorthims